sql_app/schemas.py

```diff
@@ -0,0 +1,29 @@
+"""Schemas for common attributes while creating or reading data."""
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class IdeaBase(BaseModel):
+    title: str
+    description: Optional[str] = None
+    date: datetime
+    # TODO: image: file --> how do I do this?
+
+    class Config:
+        """Provides configurations to pydantic.
+        The pydantic model will read data even if it is not a dict, but an ORM model."""
+        orm_mode = True
+
+
+class IdeaCreate(IdeaBase):
```

Comment:
whats this for?

Reply:
Pydantic's orm_mode will tell the Pydantic model to read the data even if it is not a dict, but an ORM model (or any other arbitrary object with attributes).

Source: https://fastapi.tiangolo.com/tutorial/sql-databases/#use-pydantics-orm_mode